### PR TITLE
Cleanup project files

### PR DESCRIPTION
**Pull request by martijn00**

### CODE REVIEW

**Comment by martijn00** [comment]
@sergeyshaykhullin Minio has been updated to 5.0.0. Can you merge this and release a 5.0.0 version as well?

File: src/Minio.AspNetCore/MinioClientFactory.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public MinioClient CreateClient(string name)
     {
       var options = optionsMonitor.Get(name);
 
-      var client = new MinioClient()
+      using var client = new MinioClient()
```
Comment (sergeyshaykhullin) [inline src/Minio.AspNetCore/MinioClientFactory.cs:23]: Does this using will dispose client and return disposed instance from CreateClient?

Comment (sergeyshaykhullin) [reply src/Minio.AspNetCore/MinioClientFactory.cs:23]: It seems like IMinioClientFactory owner is responsible to add usings from calling side

**Review by sergeyshaykhullin — CHANGES_REQUESTED** [review]
Thank you for your project cleanup

I have one question about disposing MinioClient from factory

**Comment by martijn00** [comment]
@sergeyshaykhullin done

**Comment by martijn00** [comment]
@sergeyshaykhullin fixed the test now

**Comment by martijn00** [comment]
@sergeyshaykhullin thanks! Can you push a new nuget release?

**Comment by sergeyshaykhullin** [comment]
@martijn00 There are some problems with CI

Can you please check CI changes?

**Pull request merged**